modules/terraform-aws-alternat/main.tf

```diff
@@ -40,10 +40,13 @@ locals {
   )
 
   endpoints = merge(local.ec2_endpoint, local.lambda_endpoint)
+
+  reuse_nat_instance_eips = try(length(var.nat_instance_eip_ids), 0) > 0
```

Comment:
We might want to check that the number of EIP IDs is the same length as the number of subnets. 

```
  reuse_nat_instance_eips = length(var.nat_instance_eip_ids) == length(var.vpc_public_subnet_ids)
```

Reply:
There should definitely be some sort of check for this, but the only thing I don't like about it is that `var.nat_instance_eip_ids` would be silently ignored if the wrong number of EIPs is specified. I think maybe the best thing to do is to add the conditional check you suggested, but then document that this variable is ignored if it does not match the number of subnets (in the description of `var.nat_instance_eip_ids`).